Bluepill-runner/Bluepill-runner/BPPacker.m

```diff
@@ -85,4 +88,73 @@ + (NSMutableArray *)packTests:(NSArray *)xcTestFiles
     return bundles;
 }
 
+#pragma mark - Private helper methods
+
+/*!
+
+ @brief Updates the config to expand any testsuites in the tests-to-run/skip into their individual test cases.
+ 
+ @discussion Bluepill supports passing in just the 'testsuite' as one of the tests to 'include' or 'exclude'.
+ This method takes such items and expands them out so that the 'packTests:' method above can simply
+ work with a list of fully qualified tests in the format of 'testsuite/testcase'.
+ 
+ @param config the @c BPConfiguration for this bluepill-runner
+ @param xcTestFiles an NSArray of BPXCTestFile's to retrieve the tests from
+ @return an updated @c BPConfiguration with testCasesToSkip and testCasesToRun that have had testsuites fully expanded into a list of 'testsuite/testcases'
+
+ */
++ (BPConfiguration *)normalizeBPConfiguration:(BPConfiguration *)config
+                                withTestFiles:(NSArray *)xcTestFiles {
+    
+    config = [config mutableCopy];
```

Comment:
Ok, so it is `bp` that is missing the testsToRun. Will refactor so that `bp` can get this too.